dashboard/cron.py

```diff
@@ -91,7 +91,7 @@ def update_datetime_field(course_obj, course_field_name, warehouse_dataframe, wa
         logger.info(f"Update of {course_field_name} skipped; existing value was found.")
     else:
         warehouse_field_value = warehouse_dataframe[warehouse_field_name].iloc[0]
-        if warehouse_field_value is not None:
+        if not pd.isnull(warehouse_field_value):
```

Comment:
Minor but I believe this is an alias for pd.isna. Probably doesn't matter which one to use.

Also I believe Pandas has a [pd.notna](https://github.com/pandas-dev/pandas/blob/v0.23.4/pandas/core/dtypes/missing.py#L259-L336) (and pd.notnull) rather than using this not. The logic here is slightly more than just calling "not" and probably should be used.

Reply:
Yeah, that looks right, and your suggestion makes sense. I can commit that in a second.